pkg/madmin/api.go

```diff
@@ -365,6 +364,10 @@ func (adm AdminClient) executeMethod(ctx context.Context, method string, reqData
 		// Initiate the request.
 		res, err = adm.do(req)
 		if err != nil {
+			// Give up right away if it is a connection refused problem
+			if strings.HasSuffix(err.Error(), "connect: connection refused") {
```

Comment:
how about using `errors.Is(err, syscall.ECONNREFUSED)` ?

Reply:
Yes, definitely and I think it is not portable at Windows OS.
I'll change it